homeassistant/components/cisco_mobility_express/device_tracker.py

```diff
@@ -0,0 +1,82 @@
+"""Support for Cisco Mobility Express."""
+import logging
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.device_tracker import (
+    DOMAIN, PLATFORM_SCHEMA, DeviceScanner)
+from homeassistant.const import (
+    CONF_HOST, CONF_USERNAME, CONF_PASSWORD, CONF_SSL, CONF_VERIFY_SSL)
+
+
+REQUIREMENTS = ['ciscomobilityexpress==0.1.2']
+
+
+_LOGGER = logging.getLogger(__name__)
+
+DEFAULT_SSL = False
+DEFAULT_VERIFY_SSL = True
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_HOST): cv.string,
+    vol.Required(CONF_USERNAME): cv.string,
+    vol.Required(CONF_PASSWORD): cv.string,
+    vol.Optional(CONF_SSL, default=DEFAULT_SSL): cv.boolean,
+    vol.Optional(CONF_VERIFY_SSL, default=DEFAULT_VERIFY_SSL): cv.boolean,
+})
+
+
+def get_scanner(hass, config):
+    """Validate the configuration and return a Cisco ME scanner."""
+    scanner = CiscoMEDeviceScanner(config[DOMAIN])
+
+    return scanner if scanner.success_init else None
+
+
+class CiscoMEDeviceScanner(DeviceScanner):
+    """This class scans for devices associated to a Cisco ME controller."""
+
+    def __init__(self, config):
+        """Initialize the scanner."""
+        from ciscomobilityexpress.ciscome import CiscoMobilityExpress
+
+        self.controller = CiscoMobilityExpress(
+            config[CONF_HOST],
+            config[CONF_USERNAME],
+            config[CONF_PASSWORD],
+            config[CONF_SSL],
+            config[CONF_VERIFY_SSL])
+
+        self.last_results = {}
+        self.success_init = self.controller.is_logged_in()
```

Comment:
Check this before instantiating the scanner.

Reply:
Done.